doc/build-windows.md

```diff
@@ -44,6 +44,32 @@ To install WSL on Windows 10 with Fall Creators Update installed (version >= 162
   * Open a cmd prompt and type "Ubuntu1804"
   * Create a new UNIX user account (this is a separate account from your Windows account)
 
+### Install on non-system drive (D:\ , F:\ ...)
```

Comment:
> Sorry but I don't think this section is required in Bitcoin Core docs. This can be helpful either in some Microsoft support page or Stackexchange/Stackoverflow Q&A.

I do agree with @prayank23 because the added instructions are not Bitcoin Core specific.

Reply:
I'm also not very keen on adding a lot of general instructions here, it could easily go out of date if Microsoft changes something upstream. I wonder if we should have WSL install instructions *at all*, isn't there some document on the official Microsoft site we can link to instead?